### DISCUSSION TITLE
Add grid customization

### DISCUSSION THREAD

**Discussion by jcobb88** [matoos32/jfreechart-builder#55] [Ideas]
Currently the grid can be very/too strong, especially when you change the background color. A customization method in ChartBuilder would be helpful.

**Comment by matoos32** [comment]
Thanks again for the input. Will have a look when I'm able to.

**Comment by matoos32** [comment]
Hello @jcobb88, I added configuring the major and minor gridlines as of [pull request #65](https://github.com/matoos32/jfreechart-builder/pull/65).

Hope this helps!
